template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/kudu/KuduServiceConfigProviderTest.java

```diff
@@ -39,10 +39,10 @@ public void testRangerPluginServiceConfig7211() {
         templateProcessor.setCdhVersion("7.2.11");
         TemplatePreparationObject preparationObject = getTemplatePreparationObject(templateProcessor, "7.2.11");
 
-        List<ApiClusterTemplateConfig> serviceConfigs = subject.getServiceConfigs(templateProcessor, preparationObject);
+        List<ApiClusterTemplateConfig> roleConfigs = subject.getRoleConfigs("KUDU_MASTER", preparationObject);
 
-        assertEquals(1, serviceConfigs.size());
-        assertTrue(serviceConfigs.contains(config(RANGER_KUDU_PLUGIN_SERVICE_NAME, GENERATED_RANGER_SERVICE_NAME)));
+        assertEquals(1, roleConfigs.size());
+        assertTrue(roleConfigs.contains(config(RANGER_KUDU_PLUGIN_SERVICE_NAME, GENERATED_RANGER_SERVICE_NAME)));
     }
 
```

Comment:
As discussed in Slack, please add some test cases for `isConfigurationNeeded()` (one positive and one negative at least).